app/controllers/trips_controller.rb

```diff
@@ -19,5 +19,6 @@ def dashboard
     @most_ends = Station.most_ending_rides
     @rides_by_date = Trip.rides_by_date
     @bike_metrics = Trip.bike_metrics
+    @subscription_metrics = Trip.subscription_metrics
```

Comment:
@iandouglas Should I do Trips.all and then call the class methods on the collection in the view? Or should I build the queries here in the controller and pass the results to the view?